README.md

```diff
@@ -88,31 +88,42 @@ At this point you should make a backup of the `.charon/validator_keys` folder as
 
 If taking part in the official Athena testnet, one cluster member will have to submit the `cluster-lock` and `deposit-data` files to the Obol Team, setting the stage for activation.
 
-## Step 4. Start the Distributed Validator Cluster
+## Step 4. Start the Distributed Validator Node
 
-With the DKG ceremony over, the last phase before activation is to prepare your node for validating over the long term. This repo is configured to sync an execution layer client (`geth`) and a consensus layer client (`lighthouse`).
+With the DKG ceremony over, the last phase before activation is to prepare your node for validating over the long term.
 
-**NOTE**: Update the `$BEACON_NODE_ENDPOINTS` in the `charon-config.yaml` with your beacon node URL.
+**NOTE:**
+- Update the `$BEACON_NODE_ENDPOINTS` in the `charon-config.yaml` with list of beacon nodes endpoints.
+- Update `$NAMESPACE` in all yaml files in the `manifests` directory (`manifests/*.yaml`).
 
+### Create namespace
+Choose a unique namespace name to avoid conflicts with the existing namespaces.
 ```
-# Create charon namespace
-kubectl create namespace charon
-
-# Populate .charon folder artefacts as kubernetes secrets
+NAMESPACE=<cluster_name>
```

Comment:
```suggestion
NAMESPACE=charon-k8s-dv-node
```

Reply:
Let's keep cluster_name as a placeholder otherwise everyone runs these instructions may end up using the same namespace within the same k8s cluster

Reply:
let's leave a note there then: "Please ensure you replace <cluster_name> with the name of your cluster"